engine/executor/server.go

```diff
@@ -268,6 +268,15 @@ func precheckMasterMeta(
 func convertMakeTaskErrorToRPCError(
 	register registry.Registry, err error, tp frameModel.WorkerType,
 ) error {
+	if pkgClient.ErrCreateWorkerTerminate.Is(err) {
+		return rpcerror.ToGRPCError(err)
+	}
+
+	switch tp {
+	case frameModel.DMJobMaster:
+		err = errors.ToDMError(err)
+	default:
+	}
```

Comment:
Can remove this code snippet, since it is contained in `checkBusinessErrorIsRetryable` at L 280.